package.json

```diff
@@ -8,7 +8,7 @@
   "license": "ISC",
   "scripts": {
     "clean": "rm dist/bundle.js",
-    "test": "mocha './{,!(node_modules)/**/}*.spec.js'",
+    "test": "DATABASE_URL=postgres://localhost/shoppadog_test mocha './{,!(node_modules)/**/}*.spec.js'",
```

Comment:
put on .env ?

Reply:
this is a little tricky with dotenv... i'll make a separate ticket for this one